core-api/src/test/resources/config/valid-project-config-v4.json

```diff
@@ -34,6 +35,10 @@
     {
       "id": "58339410",
       "key": "nationality"
+    },
+    {
+      "id": "583394100",
+      "key": "$opt_test"
```

Comment:
Just a recommendation. 
For a new dev, $opt_test may seem some valid reserved attribute. The key should say that i am rubbish. Like $opt_xyz $opt_007